JASP-Desktop/html/js/jaspwidgets.js

```diff
@@ -82,22 +82,23 @@ JASPWidgets.Encodings = {
 		xhr.send();
 	}
 }
+
 JASPWidgets.ExportProperties = {
 
 	format: {
 		raw: 0,
-		html: 1,
+		html: 1
 	},
 
 	process: {
 		copy: 0,
-		save: 1,
+		save: 1
 	},
 
 	imageFormat: {
 		temporary: 0,
 		resource: 1,
-		embedded: 2,
+		embedded: 2
```

Comment:
you could use string literals here, or some "consts". as it is i'm not sure what 0, 1, or 2 correspond to. just a suggestion.